test/e2e/resize_nodes.go

```diff
@@ -329,22 +329,8 @@ func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replica
 		Failf("Couldn't get the external IP of host %s with addresses %v", node.Name, node.Status.Addresses)
 	}
 	By(fmt.Sprintf("block network traffic from node %s to the master", node.Name))
-
-	// TODO marekbiskup 2015-06-19 #10085
-	// The use of MasterName will cause iptables to do a DNS lookup to
-	// resolve the name to an IP address, which will slow down the test
-	// and cause it to fail if DNS is absent or broken.
-	// Use the IP address instead.
-
-	destination := testContext.CloudConfig.MasterName
-	if providerIs("aws") {
-		// This is the (internal) IP address used on AWS for the master
-		// TODO: Use IP address for all clouds?
-		// TODO: Avoid hard-coding this
-		destination = "172.20.0.9"
-	}
-
-	iptablesRule := fmt.Sprintf("OUTPUT --destination %s --jump DROP", destination)
+	iptablesRule := fmt.Sprintf("OUTPUT --destination %s --jump DROP",
```

Comment:
I think it's simply a different assumption. We used to assume that `KUBE_MASTER` is set; now, we assume that `KUBE_MASTER_IP` is set (which works for more providers).

Did you want me to add a check at the beginning of the test for this? It would be a fail-safe for future situations / providers that don't have `KUBE_MASTER_IP` set, so would it be all right to do in a future PR if so?